packages/react-table/src/components/Table/utils/decorators/compoundExpand.tsx

```diff
@@ -32,9 +31,14 @@ export const compoundExpand: ITransform = (
   return {
     className: css(styles.tableCompoundExpansionToggle, props.isOpen && styles.modifiers.expanded),
     children: props.isOpen !== undefined && (
-      <Button variant="link" onClick={onToggle} aria-expanded={props.isOpen} aria-controls={props.ariaControls}>
+      <button
+        className={css(styles.tableButton)}
+        onClick={onToggle}
+        aria-expanded={props.isOpen}
+        aria-controls={props.ariaControls}
+      >
         {title}
-      </Button>
+      </button>
```

Comment:
Curious as to why we cannot use our own button here?

Reply:
This is probably a better question for @mattnolting since it's more css related. but we are using a `.pf-c-table__button` instead of `.pf-c-button` in the case of the compound expansion toggle.

Reply:
@dlabrecq Although `pf-c-table__button` shares some visual styling with `pf-c-button`,  it's behavior is very different in `sort` and `compound-expansion__toggle` contexts.